src/Resources/skeleton/authenticator/LoginFormAuthenticator.tpl.php

```diff
@@ -21,27 +23,43 @@
 
 class <?= $class_name; ?> extends AbstractFormLoginAuthenticator
 {
-    use TargetPathTrait;
-
-<?= $user_is_entity ? "    private \$entityManager;\n" : null ?>
+<?= $user_is_entity ? "    /** @var EntityManagerInterface */" . PHP_EOL . "    private \$entityManager;" . PHP_EOL . PHP_EOL : null ?>
```

Comment:
The `@var` annotations in these properties and the `__cosntruct()` method's `@param` are not needed because constructor arguments are type-hinted and IDEs can infer the types.